src/main/java/model/structures/Node.java

```diff
@@ -0,0 +1,119 @@
+package model.structures;
+
+import java.util.Objects;
+
+public class Node {
+    private int residentsNumber;
+    private int illnessCases;
+    private int healthyNumber;
+    private boolean isVaccinated;
+    private int infectingParameter;
+    private int id;
+    private int visitIteration;
+
+    public Node(int id, int residentsNumber, int infectingParameter) {
+        this.residentsNumber = residentsNumber;
+        this.illnessCases = 0;
+        this.healthyNumber = residentsNumber - illnessCases;
```

Comment:
Nie lepiej zrobić to jako geter do healthy number?

Reply:
getter odpowiada za czytanie wartości, a tu ją liczymy

Reply:
w sensie zrób jak chcesz ale uważaj na błędy kiedy tego nie uaktualniasz